daly-bms-uart.h

```diff
@@ -1,11 +1,10 @@
 #ifndef DALY_BMS_UART_H
 #define DALY_BMS_UART_H
 
-// Uncomment the below #define to enable debugging print statements.
-// NOTE: You must call Serial.being(<baud rate>) in your setup() for this to work
+#define XFER_BUFFER_LENGTH 13
```

Comment:
These changes should be rolled back, the comment that was removed still applies to DALY_BMS_DEBUG

Reply:
yes, my foul